boards/xtensa/intel_adsp_cavs25/board.cmake

```diff
@@ -1,7 +1,6 @@
 # SPDX-License-Identifier: Apache-2.0
 
-board_set_flasher_ifnset(misc-flasher)
-board_finalize_runner_args(misc-flasher)
+set(SUPPORTED_EMU_PLATFORMS adsp)
```

Comment:
but this is not an emulated platform.

Reply:
Thanks for your review, @nashif , got it. working on not to use emu.

Reply:
this is not a simulator, but I think you can extend the current BinaryHandler, make it supports user to invoke their own custom script to load any expected binary files(elf, bin and so on) to run their tests, like what we are doing using west-flash for custom flash runner.